packages/zk-eddsa-event-ticket-pcd/src/ZKEdDSAEventTicketPCD.ts

```diff
@@ -312,15 +314,18 @@ function snarkInputForProof(
     revealTicketIsRevoked: fieldsToReveal.revealIsRevoked ? "1" : "0",
     ticketCategory: ticketAsBigIntArray[8].toString(),
     revealTicketCategory: fieldsToReveal.revealTicketCategory ? "1" : "0",
-    reservedSignedField1: "0",
-    // These fields currently do not have any preset semantic meaning, although the intention
-    // is for them to convert into meaningful fields in the future. We are reserving them now
+    // Now used for attendee email:
```

Comment:
This comment should give some explanation of the reserved name, which might just be a "see comment below for purpose of reserved fields".